packages/components/pagination/src/pagination.ts

```diff
@@ -235,6 +237,22 @@ export default defineComponent({
       isAbsent(props.defaultCurrentPage) ? 1 : props.defaultCurrentPage
     )
 
+    const debounceMicroTask = (fn: () => void) => {
+      let scheduled = false
+      return () => {
+        if (scheduled) return
+        scheduled = true
+        Promise.resolve().then(() => {
+          scheduled = false
+          fn()
+        })
+      }
+    }
+
```

Comment:
Thank you for your contribution. 
I think no need to use debounce, Now it's trigger `change` in events of `currentChange` and `sizeChange`. If we switch the order, trigger `change` first, and then trigger `currentChange` and `sizeChange` in the `change` event, there won't be trigger multiple `change` event.